### PR TITLE
Oxygen Timeout + Prevent Exploit

### PR DESCRIPTION
**Pull request by MatthewRorke**
Gamers will be gamers, criminals robbing banks and hiding underwater with perma scuba gear was quite annoying.
Adding a small timeout was one thing. Persisting the oxygen tank's oxygen levels in the item meta prevents abuse of removing the oxygen tank, while still allowing people to be economical with their gear.

This appears to work okay for us. We haven't incorporated any fancy UI, maybe something on the qb-hud side would be nice. It's just using notifications for now.

TL;DR
- Timeout user oxygen when diving.
- Prevent user from applying oxygen tank in vehicle or water.
- Prevent user exploit /divinggear spam for infinite oxygen

### CODE REVIEW

**Comment by MatthewRorke** [comment]
I brought the branch up to date and added translations for text. I used google translate for the basics and provided the English.

**Comment by alexandree5** [comment]
@MatthewRorke first of all thanks for that, i was trying to do the same at mine, but the qb-diving atm doesnt work correctly, can you check it???

**Comment by MatthewRorke** [comment]
@alexandree5 I have updated the branch to be on track with the latest commits and I was able to test it on my side successfully.

**Comment by GhzGarage** [comment]
Linting passed, just need to resolve conflicts and will be good

**Comment by MatthewRorke** [comment]
@GhzGarage merged in
